pkgs/top-level/aliases.nix

```diff
@@ -607,6 +607,7 @@ mapAliases ({
   iops = throw "iops was removed: upstream is gone"; # Added 2022-02-06
   iproute = iproute2; # moved from top-level 2021-03-14
   ipsecTools = throw "ipsecTools has benn removed, because it was no longer maintained upstream"; # Added 2021-12-15
+  itch-setup = throw "itch-setup has benn removed, use itch instead"; # Added 2022-06-02
```

Comment:
```suggestion
  itch-setup = itch.itch-setup; # Added 2022-06-02
```

Reply:
`itch-setup` was previously wrapped by `steam-run`, but now it's not anymore since it's only supposed to be run by `itch` (which is already wrapped) and not standalone.
Doing this change would mean having to make the changes necessary to be usable standalone, either building it from source (already tried, I failed), `autoPatchelf`, or wrapping it with `steam-run` again. (Also, I would need to put the executable in `$out/bin`, except for the first option.)
All of this means making `itch-setup` a derivation (again), instead of a simple `fetchZip`.
I really don't think that it's something worth doing with `itch-setup`. Yes, it's the "official" method for installing Itch, but what's the point of installing an installer of a program (which, by the way, puts it in `$HOME`), when you could install said program directly? Especially with Nix, where you are supposed to put everything in the package store.